.github/workflows/atlantis-image.yml

```diff
@@ -56,6 +63,7 @@ jobs:
         images: |
           ${{ env.DOCKER_REPO }}
         labels: |
+          org.opencontainers.image.authors="Anubhav Mishra, Luke Kysow"
```

Comment:
This label seems no longer relevant as these authors are no longer actively maintaining the repo. I think we can omit this in the future